CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ add_executable(CaveCrawler
         src/World/Terrain/Block.cpp
         src/World/Player.cpp
         src/World/Terrain/ChunkManager.cpp
-        src/World/Terrain/WorldGenerator.cpp)
+        src/World/Terrain/WorldGenerator.cpp src/Inventory/Inventory.cpp include/Inventory/Inventory.h src/Inventory/Spell.cpp include/Inventory/Spell.h)
```

Comment:
Here, It's not your fault, Clion format it this way for you, but for the cpp list be more clear. You could remove the headers that won't compile and put each cpp on a different line for the next time.

Reply:
Done... it is weird that Clion puts the .h in the executable